docs/release-notes/may-2023.md

```diff
@@ -0,0 +1,181 @@
+
+- Semgrep OSS Engine Language Server updates (also related to [Semgrep Visual Studio Code extension](#semgrep-visual-studio-code-extension):
+    - Support for search and replace with rule patterns through semgrep/search.
+    - Language Server now notifies users of errors, and reason for crash.
+
+- Relaxed restrictions on symbolic propagation so that symbolic values survive branching statements. Now (with symbolic-propagation enabled) `foo(bar())`  matches the following code:
+    
+    ```python
+    def test():
+      x = bar()
+      if cond:
+        exit()
+      foo(x)
+    ```
+
+Semgrep OSS Engine section of release notes mentions only selected additions and changes, for specific bug fixes, see [Semgrep OSS Engine GitHub](https://github.com/returntocorp/semgrep/blob/develop/CHANGELOG.md) changelog and search for Fixed sections under each version.
+
+## Semgrep Supply Chain
+
+- Semgrep Supply Chain’s **License compliance** feature is now in beta. This feature enables developers to view the licenses of dependencies in all repositories that are scanned by Semgrep Supply Chain. Read the documentation: [License compliance](/semgrep-supply-chain/license-compliance).
+    - This feature also enables security engineers to **Block,** leave a **Comment,** or **Allow** dependencies when a PR introducing the dependency is first opened, based on the license of the dependency.
+    - License compliance supports many popular Copyleft, Weak-copyleft, and Permissive licenses. Other licenses can also be detected.
+    ![Screenshot of Semgrep Supply Chain Dependencies tab with licenses listed](/img/sc-license-scanning.png)
+- Semgrep Supply Chain is now available for free to Team tier users, subject to usage limits.
+- Improvements to Semgrep Supply Chain **Dependencies** page.
+
+## Semgrep Visual Studio Code
+
+Semgrep Visual Studio Code extension received new updates! Try to scan your code using the extension and let us now what we can improve. See more in [Semgrep Visual Studio Code extension](/extensions/semgrep-vs-code/) documentation. This extension allows you to check your code for vulnerabilities within seconds, with every line of code changed in the editor. The extension is built on Language Server Protocol, so you can hack it for other editors of your choice. The extension also has a dedicated [changelog on GitHub](https://github.com/returntocorp/semgrep-vscode/blob/master/CHANGELOG.md) so you can always see the latest updates.
+
+## Semgrep Code
+
+- A new Rule board version is available in Semgrep Code! The new Rule board page is called Policies. Switch to the new version by clicking **Try new version** in the Rule board header. Beside a facelift and many optimizations, you can now easily filter for various types of rule metadata, specific types of rules and make bulk edits which would have been hardly achievable with the old Rule board. Read more in [Policies](/semgrep-code/policies/) documentation.
```

Comment:
Maybe we should say that Rule board is being replaced by Policies, instead of calling Policies a new version of Rule board?

Reply:
Agreed, I will update the document.

Reply:
Done in https://github.com/returntocorp/semgrep-docs/pull/1059/commits/784568e27c214de62143d49546955ce0cb5efe7a